src/components/Dashboard/TaskItem/TaskItemUtils.js

```diff
@@ -1,13 +1,11 @@
 export const getTaskRef = (taskListRef, id) => {
-  const taskRef = taskListRef.child(`${id}`);
+  const taskRef = (taskListRef && Object.keys(taskListRef).length) && taskListRef.child(`${id}`);
```

Comment:
Previously this part: `taskListRef && ` wasn't there, but I caught an error once on app load in this place, as for some period of time `taskListRef` is undefined (__cannot read property 'length' of undefined__). So here I have to check if the `taskListRef` is not null or undefined and then if it has some value at all. So only if those two statements are true I take a reference of the task.